From 0da5b6f2404f8abeb542207aeea23820687e60e4 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 8 Jan 2012 14:13:23 -0500
Subject: [PATCH] configure: Add --with-distro=generic

For now we'll just get a basic NM setup for GNOME.
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7cb5dc0..955e2e4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -132,7 +132,7 @@ if test "z$with_distro" = "z"; then
 	exit 1
 else
 	case $with_distro in
-		redhat|suse|gentoo|debian|slackware|arch|paldo|frugalware|mandriva|pardus|linexa|exherbo) ;;
+		redhat|suse|gentoo|debian|slackware|arch|paldo|frugalware|mandriva|pardus|linexa|exherbo|generic) ;;
 		*)
 			echo "Your distribution (${with_distro}) is not yet supported!  (patches welcome)"
 			exit 1
-- 
1.7.6.4

